tools/ansible/tasks/wsu/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- hosts: localhost
+  vars:
+    project_root: "{{ playbook_dir }}/../../../.."
+    kubelet_location: "https://dl.k8s.io/v1.14.0/kubernetes-node-windows-amd64.tar.gz"
+    tmp_path: "/tmp/wsu"
+
+  tasks:
+    - name: Gather required files
+      block:
+
+        - name: Create directory to store files to be used later
+          file:
+            path: "{{ tmp_path }}"
+            state: directory
+
+        - name: Build WMCB
+          make:
+            target: build
+            chdir: "{{ project_root }}"
```

Comment:
Should we build this locally? What if we cut a release which has WMCB binary? Shouldn't we be downloading it from there, do you see any problem with this approach?

Reply:
I think for now we should build it locally until we have a hosting solution.

Reply:
Its definitely something we should change down the road though.

Reply:
My plan, is to include the WMCB binary as part of release(this handles the hosting), what about having a variable which tells to skip this step(something like `skip if !development mode`)

Reply:
I would like to have the latest version of the `WMCB` binary to be used. I guess this would become easier once we have the binary published as part of CI like it happens for images.

Reply:
I would like to address this in a future PR